woohoo.rb

```diff
@@ -0,0 +1,5 @@
+(1..20).reduce(&:+)
+(1..20).inject(&:+)
+
+'super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line'
+"super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line"
```

Comment:
Line is too long. [182/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.